hokusai/staging.yml

```diff
@@ -118,6 +118,10 @@ spec:
     - host: staging.artsy.net
       http:
         paths:
+          - path: /novo/
+            backend:
+              serviceName: force-web-novo-internal
+              servicePort: force-http
```

Comment:
I'm not sure if it's intended for this to happen somewhere else, but the `force-web-novo-internal` service needs to be defined (usually above in the services section)